### PR TITLE
[ARM] Re-enable armel CI

### PR DESCRIPTION
**Pull request by hqueue**
`armel` CI was disabled from #14528.

Let's reenable `armel` CI for PR, because `armel`(softfp ABI) uses different implementation compared to `armlb`(hardfp ABI).

/cc: @dotnet/arm32-contrib @BruceForstall @jashook

### CODE REVIEW

**Comment by hqueue** [comment]
@dotnet test ci please

**Comment by hqueue** [comment]
@dotnet-bot please test Ubuntu armlb Cross Debug Innerloop Build

**Comment by hqueue** [comment]
@dotnet-bot please test Tizen armel Cross Release Innerloop Build
@dotnet-bot please test Tizen armel Cross Debug Innerloop Build

**Comment by jashook** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -1136,8 +1136,13 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                             }
                         }
                         // Removing the regex will cause this to run on each PR.
-                        Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Build",
-                        "(?i).*test\\W+${os}\\W+${architecture}\\W+Cross\\W+${configuration}\\W+Build.*")
+                        if (configuration == 'Release' || configuration == 'Debug') {
```
Comment (jashook) [inline netci.groovy]: Please change this from debug and release to checked.

File: netci.groovy [hunk 2]
```diff
@@ -1136,8 +1136,13 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
...
+                            Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Innerloop Build")
```
Comment (jashook) [inline netci.groovy]: >"${os} ${architecture} Cross ${configuration} Innerloop Build"

to

>"${os} ${architecture} Cross ${configuration} Innerloop Build and Test"

**Comment by jashook** [comment]
Other than the nits lgtm

File: netci.groovy (hunk 1 above)
Comment (hqueue) [reply netci.groovy]: Updated. Thanks.

File: netci.groovy (hunk 2 above)
Comment (hqueue) [reply netci.groovy]: Updated. Thanks :)

**Comment by hqueue** [comment]
Update the PR with review feedback.

**Comment by jashook** [comment]
@dotnet-bot test ci please

**Pull request merged**